qa/tasks/ceph_manager.py

```diff
@@ -1018,7 +1018,8 @@ def do_thrash(self):
                     self.ceph_manager.raw_cluster_cmd('osd', 'reweight',
                                                       str(osd), str(1))
                 if random.uniform(0, 1) < float(
-                        self.config.get('chance_test_map_discontinuity', 0)):
+                        self.config.get('chance_test_map_discontinuity', 0)) \
+                        and len(self.in_osds) > 4:  # kludge: avoid m=2,k=2 stall
```

Comment:
Hmm, actually from my reading of the test I think we need to look at up_osds rather than in_osds? Since ```test_map_discontinuity``` waits for things to go clean.